explore-cardano/understanding-the-delegation-mechanism.md

```diff
@@ -1,5 +1,34 @@
 # Understanding the Delegation Mechanism
 
-As Cardano is a proof-of-stake system, owning ada not only allows you to buy goods or services, but also confers upon you the right and obligation to participate in the protocol and create blocks. 
+The stake delegation is a mechanism added to the Cardano proof of stake (PoS) protocol that allows the protocol to scale even in a setting where the set of stakeholders is highly fragmented.
 
-These two uses can be separated by the delegation mechanism, meaning someone who owns ada can keep the spending power, while delegating the power to participate in the protocol to someone else, a stake pool. It is important to note that funds can be spent normally at any time, regardless of how they are delegated.
+Anyone who owns ada can participate in the stake delegation process while retaining their spending power, while simultaneously delegating the power to participate in the protocol to a stake pool. Note that you can spend your ada normally at any time, regardless of how you delegated it. This mechanism will enable stakeholders to delegate their “voting rights” i.e., the right to participate in the slot leader election  process in each epoch.
```

Comment:
> Anyone who owns ada can participate in the stake delegation process while retaining their spending power, while simultaneously delegating the power to participate in the protocol to a stake pool.

It sounds like we're saying the same thing twice: "Anyone who onws ada can do A, while doing B, while simultaneously doing A."

Lets either not repeat it, or clarify a distinction if a distinction is intended.